easybuild/easyconfigs/v/VisPy/VisPy-0.14.1-foss-2023a.eb

```
@@ -0,0 +1,44 @@
+easyblock = 'PythonBundle'
+
+name = 'VisPy'
+version = '0.14.1'
+
+homepage = 'https://vispy.org'
+description = """VisPy is a high-performance interactive 2D/3D data visualization library
+ leveraging the computational power of modern Graphics Processing Units (GPUs) through the
+ OpenGL library to display very large datasets."""
+
+toolchain = {'name': 'foss', 'version': '2023a'}
+
+dependencies = [
+    ('Python', '3.11.3'),
+    ('SciPy-bundle', '2023.07'),
+    ('PyQt5', '5.15.10'),
+    ('matplotlib', '3.7.2'),  # for kiwisolver
+    ('freetype', '2.13.0')
+]
+
+use_pip = True
+
+local_inject_version = """sed -i "s/use_scm_version/version='%(version)s', &/g" setup.py &&"""
+
+exts_list = [
+    ('freetype_py', '2.4.0', {
+        'preinstallopts': local_inject_version,
+        'modulename': 'freetype',
+        'sources': ['freetype-py-2.4.0.zip'],
+        'checksums': ['8ad81195d2f8f339aba61700cebfbd77defad149c51f59b75a2a5e37833ae12e'],
+    }),
+    ('hsluv', '5.0.3', {
+        'checksums': ['2586bcb61d29d76e89e563a6836df24d86939961c9657f129a59f7617de45377'],
+    }),
+    ('vispy', version, {
+        'preinstallopts': local_inject_version,
+        'checksums': ['249a50979fc00a8b65109283354dcf12cf415c1a5dcf9821e113f6e590b9b93c'],
+        'use_pip': True
+    }),
+]
```

@ThomasHoffmann77 this workaround shouldn't be needed since we moved `setuptools_scm` into the main `Python` easyconfig
```suggestion
exts_list = [
    ('freetype_py', '2.4.0', {
        'modulename': 'freetype',
        'sources': ['freetype-py-2.4.0.zip'],
        'checksums': ['8ad81195d2f8f339aba61700cebfbd77defad149c51f59b75a2a5e37833ae12e'],
    }),
    ('hsluv', '5.0.3', {
        'checksums': ['2586bcb61d29d76e89e563a6836df24d86939961c9657f129a59f7617de45377'],
    }),
    ('vispy', version, {
        'checksums': ['249a50979fc00a8b65109283354dcf12cf415c1a5dcf9821e113f6e590b9b93c'],
        'use_pip': True
    }),
]
```